libkbfs/cr_chains.go

```diff
@@ -799,3 +802,66 @@ func (ccs *crChains) changeOriginal(oldOriginal BlockPointer,
 	}
 	return nil
 }
+
+// getPaths returns a sorted slice of most recent paths to all the
+// nodes in the given CR chains that were directly modified during a
+// branch, and which existed at both the start and the end of the
+// branch.  This represents the paths that need to be checked for
+// conflicts.  The paths are sorted by descending path length.  It
+// uses the corresponding node cache when looking up paths, which must
```

Comment:
Looks like we got rid of any users that passed in different `nodeCache` and `localNodeCache` instances, so I'm just going to remove the parameter.